src/Widgets/DataPersonCardWidget/DataPersonCardWidgetComponent.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  provideComponent,
+  WidgetTag,
+  // @ts-expect-error TODO: remove once officially released
+  useDataScope,
+  DataScope,
+  connect,
+  DataItem,
+  ContentTag,
+} from 'scrivito'
+
+import { DataBinaryImage } from '../../Components/DataBinaryImage'
+import { ensureString } from '../../utils/ensureString'
+import { isDataBinary } from '../../utils/dataBinaryToUrl'
+import { DataPersonCardWidget } from './DataPersonCardWidgetClass'
+import { EditorNote } from '../../Components/EditorNote'
+import personCircle from '../../assets/images/person-circle.svg'
+
+provideComponent(DataPersonCardWidget, ({ widget }) => {
+  const dataScope: DataScope | undefined = useDataScope()
+
+  if (!dataScope) {
+    return <EditorNote>No data found. Please select a data source.</EditorNote>
+  }
+
+  if (dataScope.isEmpty()) return <EditorNote>Data is empty.</EditorNote>
+
+  return (
+    <WidgetTag>
+      <ContentTag
+        content={widget}
+        attribute="headline"
+        tag="h6"
+        className="h6 text-uppercase"
+      />
+      {dataScope.take().map((dataItem) => (
+        <PersonCard dataItem={dataItem} key={dataItem.id()} />
+      ))}
+    </WidgetTag>
+  )
+})
+
+const PersonCard = connect(function PersonCard({
+  dataItem,
+}: {
+  dataItem: DataItem
+}) {
+  const name = ensureString(dataItem.get('name'))
+  const position = ensureString(dataItem.get('position'))
+  const email = ensureString(dataItem.get('email'))
+  const image = dataItem.get('image') || { url: personCircle }
```

Comment:
* [ ] FYI Conceptually this (still) contradicts the idea of enabling editors to build flexible widgets for any kind of data source. I assume this is just a step towards the final solution.